spec/dummy/spec/helpers/react_on_rails_helper_spec.rb

```diff
@@ -34,4 +34,40 @@
       expect(sanitized).to eq(hash_sanitized)
     end
   end
+  describe "#react_component" do
+    subject { react_component("App") }
+
+    let(:react_component_div) do
+      "<div id=\"App-react-component-0\"></div>"
+    end
+
+    it { expect(self).to respond_to :react_component }
+
+    it { is_expected.to be_an_instance_of ActiveSupport::SafeBuffer }
+    it { is_expected.to start_with "<script>" }
+    it { is_expected.to end_with "</div>\n\n" }
+    it { is_expected.to include react_component_div }
+
+    context "with 'id' option" do
+      subject { react_component("App", {}, id: id) }
+
+      let(:id) { "shaka_div" }
+
+      it { is_expected.to include id }
+      it { is_expected.not_to include react_component_div }
+    end
+  end
+
+  describe "#server_render_js" do
```

Comment:
These tests were originally calling just `#render_js` and got method missing, so I assumed they meant `#server_render_js`.